### ISSUE TITLE
tile shifted to the last after fullscreen (F11) when subTabbed is used

### ISSUE THREAD

**Issue by ixzh** [xmonad/xmonad-contrib#523]
### Problem Description

tile shifted to the last in order after fullscreen (F11) when subTabbed is used

### Steps to Reproduce

In spiral layout (or any others), put chrome in the main tile, press F11 to go full screen, and press F11 again to exit, Chrome window will be shifted to the last position, which is a very small window.

### Configuration File

```haskell
myLayout =
            avoidStruts .
            spacingRaw True (Border i i i i) True (Border i i i i) True .
            mkToggle (NBFULL ?? NOBORDERS ?? EOT) .
            mkToggle (single REFLECTX) .
            mkToggle (single REFLECTY) .
            hiddenWindows .
            windowNavigation .
            subTabbed . -- if commented out, full screen window restores to its spot
            screenCornerLayoutHook $
            myDefaultLayout
```

### Checklist

  - [X] I've read [CONTRIBUTING.md](https://github.com/xmonad/xmonad/blob/master/CONTRIBUTING.md)

  - [X] I tested my configuration with [xmonad-testing](https://github.com/xmonad/xmonad-testing)

**Comment by slotThe** [comment]
Sounds like this is fixed by #407, if you want to give that pr a try

**Comment by ixzh** [comment]
> Sounds like this is fixed by #407, if you want to give that pr a try

 #407 is what I'm looking for... Thank you!
